introduction_to_amazon_algorithms/jumpstart-foundation-models/text-generation-llama-2-chat.ipynb

```diff
@@ -0,0 +1,383 @@
+{
```

Comment:
<p>How about run llama chat assistant models in sagemaker jumpstart </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/aws/amazon-sagemaker-examples/pull/4333/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='0'/>

Reply:
<p>Planning to change to these:</p><ol><li>Text completion: Run Llama 2 models in SageMaker JumpStart</li><li>Chat completion: Run Llama 2 models in SageMaker JumpStart</li></ol>